docroot/modules/custom/layout_builder_custom/layout_builder_custom.module

```diff
@@ -629,6 +631,26 @@ function _layout_builder_custom_block_submit(array &$form, FormStateInterface $f
           //   https://github.com/uiowa/uiowa/issues/5036
         }
       }
+
+      if ($block->getPluginId() == 'inline_block:uiowa_card') {
+        // Make sure button display field is checked if the link title field is filled in.
+        $link_title = $form_state->getValue(
+          [
+            'settings',
+            'block_form',
+            'field_uiowa_card_link',
+          ]
+        )[0]['title'];
+        if ($link_title) {
+          $form_state->setValue(
+            [
+              'settings',
+              'block_form',
+              'field_uiowa_card_button_display',
+            ], ['value' => 1]
+          );
+        };
```

Comment:
This could be moved up to below line 615, if line 615 was moved to be first in the series of case statements.

Reply:
Working on these changes.